Lib/collections/__init__.py

```diff
@@ -693,7 +693,11 @@ def update(self, iterable=None, /, **kwds):
                 if self:
                     self_get = self.get
                     for elem, count in iterable.items():
-                        self[elem] = count + self_get(elem, 0)
```

Comment:
Since `self_get` is unused now, the definition two lines above this line can be removed.